### PR TITLE
SWC-5750

### PR DESCRIPTION
**Pull request by nickgros**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.9.1",
-    "@sage-bionetworks/react-base-table": "^1.12.3",
```
Comment (nickgros) [inline package.json:12]: This is not a direct dependency.

File: package.json [hunk 2]
```diff
@@ -25,7 +24,7 @@
     "react-use": "^17.3.1",
     "sass": "^1.30.0",
     "source-map-explorer": "^2.5.2",
-    "synapse-react-client": "2.0.62",
+    "synapse-react-client": "2.1.1",
```
Comment (jay-hodgson) [inline package.json:27]: @nickgros : bump for svg support?

**Comment by jay-hodgson** [comment]
@nickgros I ran this branch locally to start up a portal (using a locally linked SRC) and ran into a Typescript compile error.
It's as if it's not using the ts config compiler options.  This worked for you though?

Failed to compile.

/portals/src/configurations/adknowledgeportal/routesConfig.ts
TypeScript error in /portals/src/configurations/adknowledgeportal/routesConfig.ts(29,100):
Cannot find module './organizations' or its corresponding type declarations.  TS2307

    27 | import routeControlWrapperProps from './routeControlWrapperProps'
    28 | import { facetAliases } from './synapseConfigs/commonProps'
  > 29 | import { organizationCardSchema, organizationDetailsPage, organizationDetailsPageLinkConfig } from './organizations'
    30 | import { fundersSql, hackathonsSql, initiativesSql, peopleSql, studiesSql } from './resources'

File: package.json (hunk 2 above)
Comment (nickgros) [reply package.json]: Done

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**